fashify/src/utils/utils.js

```diff
@@ -0,0 +1,64 @@
+import { SortBy } from "../DataReducer/constants";
+
+const union = (...arr) => {
```

Comment:
Here, function name can be more descriptive, eg: unionOfCategories. reading union, I cannot understand its union of what two objects